examples/reference/pandas/labels.ipynb

```diff
@@ -42,6 +42,23 @@
     "df.hvplot.labels(x='Longitude', y='Latitude', text='City', text_baseline='bottom', hover=False) * \\\n",
     "df.hvplot.labels(x='Longitude', y='Latitude', text='Country', text_baseline='top', hover=False)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It's also possible to provide a template string containing the names of the columns and reduce the font size."
```

Comment:
Thanks for updating the docs! The docstring of `labels` in `core.py` should also be adapted.